### DISCUSSION TITLE
Cheat suggestion: Artificial guest cap booster

### DISCUSSION THREAD

**Discussion by fidwell** [OpenRCT2/OpenRCT2#21507] [Feature requests] [answered]
For larger parks, the soft guest cap is sometimes not enough to really fill up a park. I've seen cases where people will add tiny rides underground in order to get more guests in and fill up their paths ([example](https://youtu.be/Stg7nIFxbvE?list=PLKSsRPxziyw4-_Q9j-_Hzn8pFRzPINJXA&t=472)).

It would be handy if there were an option in the cheats menu to set the guest cap manually, or have some kind of guest cap "booster", to prevent the need for building fake rides or the "large tram" cheat.

**Comment by ocalhoun6** [comment]
If not a cheat, maybe it could be made possible for a plugin to override and change the soft guest cap?

There's already a plugin that shows your soft guest cap and what's contributing to it. A future version of that plugin could offer the option to manually change the soft guest cap. (And this would also give you the ability to decrease the soft guest cap, if you want to for some reason, maybe because your park is chronically overcrowded.)

**Reply by Crazycolbster** [reply]
This is the best way to implement it in my opinion. Invisible values like the soft guest cap are kind of going into the weeds for more casual players, but those willing to add plugins are exactly the kind that would care about the soft guest cap

**Reply by Harry-Hopkinson** [reply]
I completely agree, the soft guest cap is quite a precarious value (i.e if you had the ability to set the soft guest cap via a plugin and you inputted a bad value you can mess up your park). But the plugins are generally for more advanced users so that they would fully understand what the variable does and care about it.

**Comment by MASHzj** [comment]
I would also welcome a cheat or plugin to remove the slower guest generation when above 7000 guests.

**Comment by cheweytoo** [comment]
Oh yeah! I would be very happy to see something like this be implemented.

**Comment by astraylife1** [comment]
There kind of already is a cheat to workaround low guest generation/soft guest cap. Start a marketing campaign and then enable the never ending marketing campaign cheat.

Also, there already is a plugin to create an upper limit for guest count but it doesn’t change the soft guest cap: 
https://openrct2plugins.org/plugin/R_kgDOJ1webg/MaxGuestPlugin

I for one, have the opposite problem where my parks have too many guests and I have to turn on harder to generate to stop the inflow. The plugin above unfortunately only allows you to set it at 10k. I have parks where 11k or so is the sweet spot.

**Reply by MASHzj** [reply]
I'm building a big sandbox park, without money, so marketing is not an option here. Now that reached 7000 guests the guest count doesn't increase anymore by building more rides. The only option I have left is sending large trams from time to time, but that's not really an elegant solution in my opinion.

**Reply by astraylife1** [reply]
Until there is a way to raise the soft guest cap, you could turn money on, set your funds to some large amount that makes things irrelevant and then use marketing campaigns. If all stalls and rides are free, guests don’t need an ATM. Temporary solution for you.

**Comment by tupaschoal** [comment, marked as answer]
Implemented by #21521 via plugin
